projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,222 @@
+                  }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-status"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.startOn' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.firstDate | cxDate }}</a
+                >
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.frequency' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.trigger.displayTimeTable }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.nextOrderDate' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.trigger.activationTime | cxDate }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.total' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.subTotal.formattedValue }}</a
+                >
+              </td>
+              <td (click)="openDialog()">
```

Comment:
Should the spacing of this table data call the dialog? or should it only be when clicking the cancel 'button'

Reply:
ties in to this https://github.com/SAP/spartacus/pull/8968/files#r493857104

so the answer would be do not put the click on the td since we will just call the dialog when you click cancel

Reply:
-----
another thing is a reminder since it was said long time so I did not expect you to remember. But when building features from old accelerator, we want to most of the time match the UX. 

when you cancel in the history page, it no longer shows the cancel button and either the first date or next date says cancelled (will get back to this second point once I get access again as I'm re-initializing at the moment of writing this)

Reply:
To answer above as I got my instance running, when you click cancel in order history (replenishment), you want to hide the cancel 'button' and the data under the 'next order date' column will say Cancelled